src/App.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
-import logo from "logo.svg";
-import './App.css';
-import Products from "products.jsx";
+import logo from "./logo.svg";
+import "./App.css";
+import Products from "./products.jsx";
```

Comment:
imports were wrong, you need to specify the path. Since those files are in the same folder, you can use `./`